site/src/routes/repl/[id]/_components/AppControls/index.svelte

```diff
@@ -63,6 +64,9 @@
 			const gist = await r.json();
 			dispatch('forked', { gist });
 
+			modified_count = 0;
+			repl.markSaved()
```

Comment:
```suggestion
			repl.markSaved();
```

Missing semicolon for consistency